File: accesoAFicheros/src/accesoAFicheros/escribirTextoPlano.java
package accesoAFicheros;

import java.io.FileWriter;
import java.io.PrintWriter;
import java.io.IOException;

public class escribirTextoPlano {
	
	public void escribir(String nombre, String frase) {
		
		FileWriter fichero = null;
		PrintWriter escritor = null;
		
		try {
			fichero = new FileWriter("archivo.txt");
			escritor = new PrintWriter(fichero);
			escritor.println(frase);
		} catch (IOException  ioe) {
			ioe.printStackTrace() ;
		} finally {
			if (fichero != null)
			  try {
			    fichero.close();
			  } catch (IOException  ioe) {
				  ioe.printStackTrace() ;
			  }
		}
	}
}
	

File: inicio_kotlin/src/escribirTextoPlano.kt
import java.io.FileWriter
import java.io.IOException
import java.io.PrintWriter

fun escribirTextoPlano() {
    var fichero: FileWriter? = null
    var escritor: PrintWriter? = null

    try {
        fichero = FileWriter("archivo.txt");
        escritor = PrintWriter(fichero) ;
        escritor.println("Esto es una linea del fichero");
    } catch (ioe: IOException) {
        ioe.printStackTrace() ;
    } finally {
        if (fichero != null)
            try {
                fichero.close();
            } catch (ioe: IOException) {
                ioe.printStackTrace() ;
            }
    }
}